core/elbv2.go

```diff
@@ -0,0 +1,37 @@
+package core
+
+import (
+	"github.com/aws/aws-sdk-go/service/elbv2"
+)
+
+// Returns a list of ELB (v2) - also known as ALB - based on the input from the different regions
+func (c *Connector) GetLoadBalancersV2(input *elbv2.DescribeLoadBalancersInput) ([]*elbv2.DescribeLoadBalancersOutput, error) {
```

Comment:
But as `Errs` satisfies `error`, why returning it, rather than an `error` type? As we discussed, my understanding was that it was more standard, and would simply make more sense. No?